### DISCUSSION TITLE
Issues with low growth rates and fluxes

### DISCUSSION THREAD

**Discussion by SchuShoe** [micom-dev/micom#83] [Q&A] [answered]
I'm trying to predict the microbial growth and fluxes of three metabolites using a meat-based medium.
But the growth and fluxes seemed very low.  I also tried running the simulation with high-carbs medium, but the results were similar.  
This may be a rookie question, but what should I do to obtain more realistic or biologically relevant fluxes of those three metabolites?
Here're the code and results
`from micom import load_pickle`
`from micom.qiime_formats import load_qiime_medium`
`medium = load_qiime_medium("/content/materials/media/meat.alo.qza")`
`medium.index = medium.reaction`
`com = load_pickle('/content/materials/models/ERR1883195.pickle')`
`com.medium = medium.flux`
`com`
`sol = com.cooperative_tradeoff(fraction=0.5, fluxes=True, pfba=False)
sol`
![image](https://user-images.githubusercontent.com/101419498/164894082-730ad37b-35ba-4080-9d10-ccff45668dd7.png)
![image](https://user-images.githubusercontent.com/101419498/164894151-a272a14a-04d8-48f4-b3db-da3dfbd54ba0.png)

**Comment by cdiener** [comment, marked as answer]
It's not insanely low but yeah somewhat low. Growth rates are pretty directly coupled to the nutritional content of the medium/diet. So the easiest solution is to design a diet with higher nutrient content in the lower gut. You probably already used `complete_medium` and could try with higer target growth rates there. Other approaches are:

- Increase the flux bounds for components already in the medium (diet questionnaires etc. often underestimate the carbon and nitrogen content since they only cover a small set of metabolites). For instance, you can try multiplying all bounds with a factor >1.
- Add in host-supplied metabolites such as bile acids and mucins.
- Check that not all components are efficiently adsorbed in the small intestine. You will need some complex starches or fibers, since those are the major carbon source for the gut microbiome.

**Reply by SchuShoe** [reply]
Thank you for the suggestions!
